src/main/java/com/google/sps/servlets/NewBigBusiness.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for adding new bussiness to the google datastore system*/
+@WebServlet("/new-bigBusiness")
+public class NewBigBusiness extends HttpServlet {
```

Comment:
I don't think this servlet is being used anywhere. Is it?

Reply:
no I actually made a separate branch that will use this file, but I can't really remove this from this branch. However, I took these two commits to a different branch that I should upload by the end of today

Reply:
In that case, please remove it. It's outside of the scope of this pull request.

Reply:
Will do